models/GridConfig.php

```diff
@@ -297,6 +299,22 @@ public function setSetAsFavourite($setAsFavourite)
         $this->setAsFavourite = (bool) $setAsFavourite;
     }
 
+    /**
+     * @return bool
+     */
+    public function isSaveFilters()
+    {
+        return $this->saveFilters;
+    }
+
+    /**
+     * @param bool $saveFilters
+     */
+    public function setSaveFilters($saveFilters)
+    {
+        $this->saveFilters = (bool) $saveFilters;
+    }
```

Comment:
Please use type hints
```suggestion
    public function isSaveFilters(): bool
    {
        return $this->saveFilters;
    }

    public function setSaveFilters(bool $saveFilters): void
    {
        $this->saveFilters = $saveFilters;
    }
```